app/Models/Status.php

```diff
@@ -32,7 +32,52 @@ public function event() {
     }
 
     public function getFavoritedAttribute() {
-        return !!$this->likes->where('user_id', Auth::id())->first();
+        return $this->likes->contains('user_id', Auth::id());
+    }
+
+    public function getSocialTextAttribute() {
+        $postText = trans_choice(
+            'controller.transport.social-post',
+            preg_match('/\s/', $this->trainCheckin->HafasTrip->linename),
+            [
+                'lineName'    => $this->trainCheckin->HafasTrip->linename,
+                'destination' => $this->trainCheckin->Destination->name
+            ]
+        );
+        if ($this->event !== null) {
+            $postText = trans_choice(
+                'controller.transport.social-post-with-event',
+                preg_match('/\s/', $this->trainCheckin->HafasTrip->linename),
+                [
+                    'lineName'    => $this->trainCheckin->HafasTrip->linename,
+                    'destination' => $this->trainCheckin->Destination->name,
+                    'hashtag'     => $this->event->hashtag
+                ]
+            );
+        }
+
+        if (isset($status->body)) {
+            $eventIntercept = "";
+            if ($this->event !== null) {
+                $eventIntercept = __('controller.transport.social-post-for') . '#' . $this->event->hashtag;
+            }
+
+            $appendix = " (@ " .
```

Comment:
I know that this ccode part was moved. But why not using something like https://www.php.net/strtr ? I think it would be more readable